src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+export default function operate(str1, str2, operator) {
+  const num1 = Big(parseFloat(str1));
+  const num2 = Big(parseFloat(str2));
+
+  switch (operator) {
+    case '+':
+      return (num1.plus(num2)).toString();
+    case '-':
+      return (num1.minus(num2)).toString();
+    case 'x':
+      return (num1.times(num2)).toString();
+    case '÷':
+      if (num2 == 0) {
```

Comment:
while i know it can be said that there's no life-or-death difference between `==` and `===`, linters and even chrome console raises an issue with this for a reason. Let's convert `==` to `===` please. Thank you

Reply:
In this case I have to use == instead of === because I'm comparing the object from big.js to 0, so the triple equal will return false.